src/verify.ts

```diff
@@ -83,8 +83,13 @@ export const registryVerifier: Verifier<
       const documentData = getData(document);
       return documentData.proof.method === v3.Method.DocumentStore;
     }
-    const documentData = getData(document as WrappedDocument<v2.OpenAttestationDocument>);
-    return documentData.issuers.some(issuer => "documentStore" in issuer || "certificateStore" in issuer);
+
+    if (utils.isWrappedV2Document(document)) {
```

Comment:
dont use this one because it actually doesn't test anything and I dunno how to write one that is actually helpful